### PR TITLE
Update project structure and missing files

### PR DESCRIPTION
**Pull request by alejandrodevs**
Update project structure:
- CODE_OF_CONDUCT file was added.
- MIT LICENSE file was updated with the latest version.
- CHANGELOG file was added.
- master branche references were changed to main branch.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -80,8 +80,32 @@ Optionally, if you want to do something different when authentication
...
-The gem is available as open source under the terms of the [MIT License](https://opensource.org/licenses/MIT).
+
+The gem is available as open source under the terms of the
+[MIT License](https://opensource.org/licenses/MIT).
+
+## Code of Conduct
+
+Everyone interacting in the Gamora project's codebases, issue trackers,
+chat rooms and mailing lists is expected to follow the
+[code of conduct](https://github.com/amco/gamora-rb/blob/main/CODE_OF_CONDUCT.md).
```
Comment (javierg) [inline README.md:111]: 👏

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+## [Unreleased]
+
+## [0.5.0] - 2022-09-05
+
+- Add caching support (optionally) to avoid hitting the IDP every request.
+
+## [0.4.0] - 2022-09-02
+
+- Add unauthentication endpoint to remove access/refresh tokens and
```
Comment (javierg) [inline CHANGELOG.md]: the word `unauthentication`. doesnt exists, you can use `unauthenticated`

**Review by javierg — APPROVED** [review]
:shipit:

File: CHANGELOG.md (hunk 2 above)
Comment (alejandrodevs) [reply CHANGELOG.md]: Fixed, at the end I used the word `logout`.

**Pull request merged**